css/style.css

```diff
@@ -0,0 +1,1159 @@
+/*Обнуление*/
+*{padding:0;margin:0;border:0;}
+*,*:before,*:after{-moz-box-sizing:border-box;-webkit-box-sizing:border-box;box-sizing:border-box;}
+:focus,:active{outline:none;}
+a:focus,a:active{outline:none;}
```

Comment:
Codacy found an issue: [Expected single space after "{" of a single-line block (block-opening-brace-space-after)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)